docs/stacks-guide-gcp.md

```diff
@@ -0,0 +1,424 @@
+---
+title: "Crossplane Stacks Guide: GCP Setup"
+toc: false
+weight: 421
+indent: true
+---
+
+# Crossplane Stacks Guide: GCP Setup
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Install the GCP Stack](#install-the-gcp-stack)
+3. [Configure GCP Account](#configure-gcp-account)
+4. [Configure Crossplane GCP Provider](#configure-crossplane-gcp-provider)
+5. [Set Up Network Resources](#set-up-network-resources)
+6. [Configure Provider Resources](#configure-provider-resources)
+7. [Recap](#recap)
+8. [Next Steps](#next-steps)
+
+## Introduction
+
+In this guide, we will set up a GCP provider in Crossplane so that we
+can install and use the WordPress sample stack, which depends on MySQL
+and Kubernetes!
+
+Before you begin, you will need:
+
+* Everything from the [Crossplane Stacks Guide][stacks-guide] before the
+  cloud provider setup
+  - A `kubectl` pointing to a Crossplane control cluster
+  - The [Crossplane CLI][crossplane-cli] installed
+* An account on [Google Cloud Platform][gcp]
+
+At the end, we will have:
+
+* A Crossplane control cluster configured to use GCP
+* The boilerplate of a GCP-based project spun up
+* Support in the control cluster for managing MySQL and Kubernetes
+  cluster dependencies
+* A slightly better understanding of:
+  - The way cloud providers are configured in Crossplane
+  - The way dependencies for cloud-portable workloads are configured in
+    Crossplane
+
+We will **not** be teaching first principles in depth. Check out the
+[concepts document][crossplane-concepts] for that.
+
+## Install the GCP Stack
+
+After Crossplane has been installed, it can be extended with more
+functionality by installing a Crossplane Stack! Let's install the stack
+for Google Cloud Platform (GCP) to add support for that cloud provider.
+We can use the Crossplane CLI for this operation:
+
+```
+kubectl crossplane stack install 'crossplane/stack-gcp:master' stack-gcp
+```
+
+To install to a particular namespace, you can use the `generate-install`
+command and pipe it to `kubectl apply` instead, which gives you more
+control over how the stack's installation is handled. Everything is
+Kubernetes object!
+
+Since this is an infrastructure stack, we need to specify that it's
+cluster-scoped via `--cluster` flag.
+```
+kubectl create namespace gcp
+kubectl crossplane stack generate-install --cluster 'crossplane/stack-gcp:master' stack-gcp | kubectl apply --namespace gcp -f -
+```
+
+The namespace that we install the stack to is also the one where our
+managed GCP resources will reside. When a developer requests a resource
+by creating a **resource claim** in a namespace `mynamespace`, the
```

Comment:
I believe we will want `mynamespace` to be something like `app-project1-dev`

Reply:
Should be fixed